.github/workflows/genbuilds.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env xonsh
```

Comment:
Why translate from xonsh?

Reply:
wanted to autoformat and lint.

Reply:
This is not blocking, but I would prefer that we keep as much in xonsh as possible.

Reply:
Ok do you want to rename to .xsh ? here is not any benefit using it though.

Reply:
No, its fine as is.